### ISSUE TITLE
R CMD build fails

### ISSUE THREAD

**Issue by ateucher** [bcgov/bcmaps.rdata#9]
```
$ R CMD build bcmaps.rdata
* checking for file 'bcmaps.rdata/DESCRIPTION' ... OK
* preparing 'bcmaps.rdata':
* checking DESCRIPTION meta-information ... OK
* checking for LF line-endings in source and make files and shell scripts
* checking for empty or unneeded directories
* looking to see if a 'data/datalist' file should be added
Warning in doTryCatch(return(expr), name, parentenv, handler) :
  error reading the file
* building 'bcmaps.rdata_0.2.0.tar.gz'
```

**Comment by boshek** [comment]
Turns out that this is a bug in R itself as pointed out [here](https://stat.ethz.ch/pipermail/r-package-devel/2019q1/003351.html), reported [here](https://bugs.r-project.org/bugzilla/show_bug.cgi?id=17520) and ultimately fixed for R 3.6.* such that packages of *.rdata and *.rds form no longer trip up R CMD build. 

It does however seem overly restrictive to restrict users to only 3.6.*. Rather the path of least resistance here seems to be to change the name of bcmaps.rdata. I think bcmapsdata would probably be a good name for the package though we can discuss other options. I think now it is just a matter of working through all the changes that need to happen both to this package and to bcmaps to ensure a smooth transition here.

**Comment by ateucher** [comment]
I like `bcmapsdata`, and agree that restricting the R version is overkill.
